metricbeat/mb/mb.go

```diff
@@ -199,6 +199,13 @@ type ReportingMetricSetV2 interface {
 	Fetch(r ReporterV2)
 }
 
+// ReportingMetricSetV2 is a MetricSet that reports events or errors through the
+// ReporterV2 interface. Fetch is called periodically to collect events.
+type ReportingMetricSetV2Error interface {
```

Comment:
Nit: What about calling this `ReportingMetricSetV3`?

Reply:
Oh, never mind, I see now: the `V2` refers to the `ReporterV2` being used in `Fetch()`.

Reply:
Yes, and I rather not introduce one more version :-) I hope that V1 will disappear soon and there will be just 1 (2) options.